Resources/Prototypes/Entities/Objects/Weapons/Melee/knife.yml

```diff
@@ -167,6 +170,9 @@
     sprite: Clothing/Head/Hats/brownflatcap.rsi
   - type: Clothing
     sprite: Clothing/Head/Hats/brownflatcap.rsi
+  - type: Tag
+    tags:
+    - CannonRestrict
```

Comment:
this gets inherited from `parent: BladedFlatcapGrey`
```suggestion
```

Reply:
Done.